Source/restrict.cpp

```diff
@@ -8,17 +8,19 @@
 #include "utils/file_util.h"
 #include "utils/paths.h"
 
+#include <SDL.h>
+
 namespace devilution {
 
 void ReadOnlyTest()
 {
 	const std::string path = paths::PrefPath() + "Diablo1ReadOnlyTest.foo";
-	FILE *f = FOpen(path.c_str(), "w");
-	if (f == nullptr) {
+	SDL_RWops *file = SDL_RWFromFile(path.c_str(), "w");
+	if (file == NULL) {

```

Comment:
why not nullptr?

Reply:
Cause we didn't do it in other places and I didn't feel like breaking the trend :shrug:

Reply:
Looked at it a bit more and decided to change the other cases instead.